src/backend/alembic/versions/998b82dfd78f_.py

```diff
@@ -0,0 +1,30 @@
+"""empty message
```

Comment:
Could you update the migration to be in line with this PRs changes? https://github.com/cohere-ai/cohere-toolkit/pull/552

- Eg add timestamp and description, so `2024_08_01_998..f_add_agent_is_private.py` 
- Add a description instead of `empty message`